samples/drivers/i2s/i2s_codec/Kconfig

```diff
@@ -17,6 +17,14 @@ config SAMPLE_FREQ
 	help
 	  Sample frequency of the system.
 
+config CODEC_MASTER
+	bool "Configure the selected codec device as a master device."
+	default true
+
+config I2S_MASTER
```

Comment:
It's a configuration error to enable both these options or none of them, right? Why not have just one option that specifies which clock should be used? Named for example `USE_CODEC_CLOCK` (also to avoid using terms "master" and "slave"); not enabled by default, so only when needed `CONFIG_USE_CODEC_CLOCK=y` would be added in overlays.